.github/skills/code-review/SKILL.md

```
@@ -0,0 +1,150 @@
+---
+name: code-review
+description: Project-specific checklist for reviewing pull requests to the HYXi Cloud Home Assistant integration — async/coordinator patterns, entity naming/translation keys, config-entry exception handling, device registry linkage, manifest/version sync, services.yaml drift, secure logging, exception-catch/telemetry-parsing discipline, GitHub Actions hardening, and where Home Assistant's own runtime validation catches things hand-rolled tests miss. Use whenever reviewing a pull request or diff in this repository.
+---
+
+# Code Review: HYXi Cloud Home Assistant Integration
+
+This repo is a Home Assistant custom integration. When reviewing a PR, check
+the following in addition to general correctness.
+
+## 1. Async correctness
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)